web/src/components/RemoteContent.tsx

```diff
@@ -61,6 +63,13 @@ const RemoteContent = ({ html, centered = false, smallText = false }: RemoteCont
       return
     }
     const currentSandBoxRef = sandBoxRef.current
+
+    const allElements = currentSandBoxRef.querySelectorAll('*')
+    allElements.forEach(el => {
+      const element = el as HTMLElement
+      element.style.removeProperty('color')
```

Comment:
I think this is the only way to remove the color from remote content :thinking:

Reply:
I think this is a good approach 👍 However I think we should only remove this for text set to black and not in general.
See other suggestion.